search-website/README.md

```diff
@@ -19,44 +19,51 @@ To run the sample locally, follow the steps below.
+    - YOUR-SEARCH-RESOURCE-NAME
+    - YOUR-SEARCH-ADMIN-KEY - used only in bulk import for inserting
+    - YOUR-SEARCH-QUERY-KEY - used only in Function api for querying
 - [Node.js and Git](https://nodejs.org/)
 - [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=shopathome-github-jopapa) installed
-- The [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions&WT.mc_id=shopathome-github-jopapa) installed
-- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=shopathome-github-jopapa) installed
+    - [Static Web app extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) installed
+    - [Cognitive Search extention](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecognitivesearch)
 
-### Setup
+### Download sample repository
 
-1. Clone (or Fork and Clone) this repository
-1. Open `./search-website` in Visual Studio Code. 
-1. Rename the `api/local.settings.json.rename` file to `api/local.settings.json`.
+1. In a terminal, use git to clone (or Fork and Clone) this repository to your local computer:
 
-    The `local.settings.json` file holds all of the keys that the application needs.
-    
-    For this sample, there is a search index using the [goodbooks-10k dataset](https://github.com/zygmuntz/goodbooks-10k) that that is publicly available using the connection information below. The index consists of 10,000 popular books that we'll search over in our application.
-    
-    ```json
-    {
-      "IsEncrypted": false,
-      "Values": {
-        "AzureWebJobsStorage": "",
-        "FUNCTIONS_WORKER_RUNTIME": "node",
-        "SearchApiKey": "REPLACE-WITH-YOUR-SEARCH-QUERY-KEY",
-        "SearchServiceName": "REPLACE-WITH-YOUR-SEARCH-RESOURCE-NAME",
-        "SearchIndexName": "good-books",
-        "SearchFacets": "authors*,language_code"
-      }
-    }
+    ```bash
+    git clone https://github.com/Azure-Samples/azure-search-javascript-samples
     ```
 
-## Run the app locally
+1. Open that local directory in Visual Studio Code.
+
+### Run bulk import to the Search Index
+
+1. Open the `./search-website/bulk-insert/bulk_insert_books.js`.
+1. Download the [books.csv](https://raw.githubusercontent.com/zygmuntz/goodbooks-10k/master/books.csv) to the same directory as the `bulk_insert_books.js` file.
+1. Replace the following variables in the `bulk_insert_books.js` with your own values to authenticate with the Azure Search SDK:
+
+    * YOUR-SEARCH-RESOURCE-NAME
+    * YOUR-SEARCH-ADMIN-KEY
+
+1. Open a terminal in VS Code from this directory by selecting the directory, then selecting **Open Integrated Terminal**.
+ 
+1. Run the bulk import script:
+
+    ```bash
+    node bulk_insert_books.js
```

Comment:
I get this error:
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Try the new cross-platform PowerShell https://aka.ms/pscore6

PS C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert> node bulk_insert_books.js
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'csv-parser'
Require stack:
- C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert\bulk_insert_books.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert\bulk_insert_books.js:8:15)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\heidist\\azure-search-javascript-samples\\search-website\\bulk-insert\\bulk_insert_books.js'
  ]
}
PS C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert>